java/client/src/main/java/glide/api/commands/ListBaseCommands.java

```diff
@@ -225,4 +226,25 @@ public interface ListBaseCommands {
      * }</pre>
      */
     CompletableFuture<String[]> rpopCount(String key, long count);
+
+    /**
+     * Inserts <code>element</code> in the list at <code>key</code> either before or after the <code>
+     * pivot</code> value.
+     *
+     * @see <a href="https://redis.io/commands/linsert/">redis.io</a> for details.
+     * @param key The key of the list.
+     * @param position The position to insert into - either {@link InsertPosition#BEFORE} or {@link
+     *     InsertPosition#AFTER} the <code>pivot</code>.
+     * @param pivot The reference value.
+     * @param element The new element to insert.
+     * @return The list length after a successful insert operation, <code>0</code> if the <code>key
+     *     </code> doesn't exist or <code>-1</code> if the <code>pivot</code> wasn't found.
+     * @example
+     *     <pre>{@code
+     * Long length = client.linsert("my_list", BEFORE, "World", "There").get();
```

Comment:
Nit: I stick to Java conventions for variables such as "myList".

Reply:
It is a redis key name - we use `my_list` everywhere